src/components/Sing/SequencerNote.vue

```diff
@@ -123,6 +152,26 @@ const width = computed(() => {
   const noteEndBaseX = tickToBaseX(noteEndTicks, tpqn.value);
   return (noteEndBaseX - noteStartBaseX) * zoomX.value;
 });
+// 歌詞のフォントサイズをズームにあわせて調整する
+// 最小12px, 最大16px, ノートの高さは越えない
+const lyricFontSize = computed(() => {
+  const minSize = 12;
+  const maxSize = 16;
+  // ノートの高さを見た目で越えない範囲で最大値を設定
+  const baseSize = Math.min(height.value - 6, maxSize);
+  // ズームによる拡大率を調整(計算値に特に根拠はない/見た目あわせ)
```

Comment:
（細かいですが）
コメント内の`/`の意味が曖昧かも。
多分併記してるだけだと思うので、崩してあげると誤解されにくいかも！
```suggestion
  // ズームによる拡大率を調整(計算値に特に根拠はない。見た目あわせ。)
```

Reply:
こちら修正済みです